.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,16 @@ Describe your proposed changes here.
 - [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
 - [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
 - [ ] Rebased/mergeable
+- [ ] Root cause description
+   * Describe the root cause of the bug. 
+   * Did the fix introduce the change in behavior?
+   * What was done to fix the problem?
 - [ ] Tests pass
+   - [ ] Unit tests
+   - [ ] Integration tests
+      * The buf fix affects the interface between components
```

Comment:
Hey, @gshif I see one other typo (`buf` instead of `bug`). Somehow I missed this from before.